chia/full_node/full_node.py

```diff
@@ -1566,7 +1566,26 @@ async def respond_block(
         self.full_node_store.clear_unfinished_blocks_below(clear_height)
         if peak.height % 1000 == 0 and not self.sync_store.get_sync_mode():
             await self.sync_store.clear_sync_info()  # Occasionally clear sync peer info
-        self._state_changed("block")
+
```

Comment:
Actually I think this can all go into peak_post_processing_2 at the bottom of it, since it is kind of post processing related

Reply:
Discussed in keybase - the `ReceiveBlockResult` isn't available in post processing 2, so we're leaving here for now